db.py

```diff
@@ -16,9 +16,9 @@ class Role(Enum):
 class User(db.Model):
     __tablename__ = 'user'
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String, nullable=True)
-    first_name = db.Column(db.String, nullable=True)
-    email = db.Column(db.String, nullable=True, unique=True)
+    name = db.Column(db.String(30), nullable=True)
+    first_name = db.Column(db.String(30), nullable=True)
+    email = db.Column(db.String(100), nullable=True, unique=True)
```

Comment:
```suggestion
    name = db.Column(db.String(100), nullable=True)
    first_name = db.Column(db.String(100), nullable=True)
    email = db.Column(db.String(200), nullable=True, unique=True)
```

Don't be stingy: You can set a length of 100 for both so that there won't be any issue in case of aristocratic names.